squashfs-tools/xz_wrapper.c

```diff
@@ -521,13 +617,23 @@ static void xz_usage(FILE *stream)
 	fprintf(stream, " header as either 2^n or as 2^n+2^(n+1).\n\t\t");
 	fprintf(stream, "Example dict-sizes are 75%%, 50%%, 37.5%%, 25%%, or");
 	fprintf(stream, " 32K, 16K, 8K\n\t\tetc.\n");
+	fprintf(stream, "\t  -Xpreset <preset-level or extreme>\n");
```

Comment:
nit: remove extra space, and add valid number range.
```suggestion
	fprintf(stream, "\t  -Xpreset <preset-level>\n");
	fprintf(stream, "\t\tUse <preset-level> as the XZ preset to use on");
	fprintf(stream, " compress.\n\t\t<preset-level> should be 0 .. 9");
	fprintf(stream, " (default 6), or \"extreme\".\n");
```